demo/index.html

```diff
@@ -72,15 +72,31 @@
       `,
       label: 'Share this item',
       id: 'share',
+      title: 'Share this item',
       component: html`<ia-sharing-options
         identifier="goody"
         type="book"
         creator="Welsh, Charles"
         description="The history of Little Goody Two-Shoes : otherwise called Mrs. Margery Two-Shoes ... [1766 edition]"
-        baseHost="www-shaner.archive.org"
-      ></ia-sharing-options>`
+        baseHost="www-shaner.arc,hive.org"
```

Comment:
You've got a comment in the attribute value that should be removed.

Reply:
ha! I thought that was something clever you put in 😅